ChiantiPy/tools/util.py

```diff
@@ -457,10 +458,11 @@ def scale_bti(evin,crossin,f,ev1):
     References
     ----------
     .. [1] Burgess, A. and Tully, J. A., 1992, A&A, `254, 436 <http://adsabs.harvard.edu/abs/1992A%26A...254..436B>`_
+    .. [2] Dere, K. P., 2007, A&A, 466, 771, '<http://adsabs.harvard.edu/abs/2007A%26A...466..771D>'
```

Comment:
the link needs to be inside backticks, e.g.`` and be followed by an underscore outside of the backticks. See Burgess and Tully reference above.